web_app_certos/src/views/graphiql.tsx

```diff
@@ -0,0 +1,60 @@
+
+import { useEffect } from "react";
+import { useSafeSetState } from "react-admin";
+import { Box } from "@mui/material";
+import { GraphiQL } from "graphiql";
+import { createGraphiQLFetcher } from '@graphiql/toolkit';
+import { getRawAuthorization } from "../components/auth_provider";
+import type { GraphQLSchema } from "graphql";
+import { envs } from "../components/cypher";
+import Loading from "./loading";
+import 'graphiql/graphiql.css';
+
+
+const Graphiql = () => {
+  const [schema, setSchema] = useSafeSetState<GraphQLSchema>();
+  const [body, setBody] = useSafeSetState<string | boolean>();
+  const [headers, setHeaders] = useSafeSetState<string>("");
+  const [headersSetted, setHeadersSetted] = useSafeSetState<boolean>(false);
```

Comment:
Let's not use headersSetted, just use headers and set it to an empty string when not set.

Reply:
Done